src/pages/starter-kit/checkout/configure.md

```diff
@@ -101,24 +101,8 @@ This script must finish running before you deploy the application for event regi
 
```

Comment:
@obarcelonap Actually the eventing parts are pretty necessary to Payments, isn't it? Could you check if keeping the eventings in the common section makes more sense than moving them under Payments?
https://adobedocs.github.io/commerce-extensibility/starter-kit/checkout/configure/#scripts
https://adobedocs.github.io/commerce-extensibility/starter-kit/checkout/use-cases/#third-party-events-processing

@oshmyheliuk are they also used in Shipping?

Reply:
No, eventing is not used for shipping, only webhooks.

Reply:
I believe it is ok to have eventing/webhooks sections in shared part. At the end these are oope tools to make the work. On each concept we can have specific section related to webhooks/eventing if something else has to be explained.